core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -658,19 +661,19 @@ public StartupConsistencyCheck getStartupConsistencyCheck() {
    * @throws AccessControlException if permission checking fails
    */
   public long getFileId(AlluxioURI path) throws AccessControlException {
-    long flushCounter = AsyncJournalWriter.INVALID_FLUSH_COUNTER;
+    Mutable<Long> flushCounter = new MutableObject<>(AsyncJournalWriter.INVALID_FLUSH_COUNTER);
```

Comment:
Alternatively,  instead of introducing this `Mutable<Long>` wrapper, we could create a class  `FlushCounter` as the wrapper.  One advantage of doing so is that we could have `FlushCounter` a `Closable` resource, which runs ` waitForJournalFlush` automatically in its `close` method. In this way, the code becomes:

```java
try (LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.WRITE);
    FluchCounter counter = new FlushCounter()) {
  // permission check
  // run xxxAndJournal(inodePath, counter, other argus)
} 
// you don't need a finally any more.
```
I slightly incline to `FlushCounter` approach as it looks cleaner to me, but not strong on this. Leave this decision to you.